### DISCUSSION TITLE
Windows/Linux plugin

### DISCUSSION THREAD

**Discussion by kostaa** [espresso3389/flutter_pdf_render#38] [General]
I am looking for a replacement pdf library. The one that I am currently using in my app does not seem to be maintained anymore . I was wondering if you are planning to add support for Windows and maybe Linux as well.

**Comment by espresso3389** [comment]
Currently, the plugin depends on platform specific PDF renderer implementations. It uses [CGPDFDocument](https://developer.apple.com/documentation/coregraphics/cgpdfdocument) on iOS and [PdfRenderer](https://developer.android.com/reference/android/graphics/pdf/PdfRenderer) on Android.

For macOS, the code is almost identical to iOS and I did support it easily.

For Windows and Linux, they don't have PDF rendering engine out of box. So I'm now working on [a rendering engine](https://github.com/espresso3389/sobar_pdf) for the platforms.

I created issue #40 for the task anyway.

**Reply by kostaa** [reply]
That's great news!
